YangMingShan/YMSPhotoPicker/Public/YMSPhotoPickerViewController.m

```diff
@@ -419,7 +419,7 @@ - (void)updateViewWithCollectionItem:(NSDictionary *)collectionItem
     
     UIButton *albumButton = [UIButton buttonWithType:UIButtonTypeSystem];
     albumButton.tintColor = self.theme.titleLabelTextColor;
-    albumButton.titleLabel.font = [UIFont systemFontOfSize:18.0];
+    albumButton.titleLabel.font = self.theme.titleLabelFont ?: [UIFont systemFontOfSize:18.0];
```

Comment:
@PLJNS I forgot to change this line. Could you help me to change this line into 
`albumButton.titleLabel.font = self.theme.titleLabelFont;`
? Because theme.titleLabelFont is non-null property, we don't need to check it here.